packages/backend/README.md

```diff
@@ -0,0 +1,40 @@
+![GitHub license](https://img.shields.io/badge/license-Apache_2.0-blue.svg)
+
+# Application Wizard
+
+![](screenshot.png)
+
+With the Application Wizard extension, you can benefit from a rich user experience for yeoman generators. This extension allows developers to reuse existing yeoman generators and provide wizard-like experience with no development efforts.
```

Comment:
put the short description text before the screenshot?

Reply:
Thank you so much for all your comments and support while implementing this task. I fixed partially. Team decided to merge and do the rest of fixes in separate change.